app/models/concerns/project_media_getters.rb

```diff
@@ -153,7 +153,7 @@ def get_title
       return self.send(title_mapping[title_field]).to_s
     end
     title = self.original_title
-    [self.analysis['file_title'], self.analysis['title'], self.fact_check_title, self.claim_description_content].each do |value|
+    [self.analysis['file_title'], self.analysis['title'], self.fact_check_title(true), self.claim_description_content].each do |value|
```

Comment:
I think for `self.fact_check_title(true)`  we should not call cached field with true to enforce regenerate cached field and make sure that cached field updated in relevant actions

Reply:
There was a race condition so I had to force that refresh... the event was triggering a call to this method, so it would fail.